hi-ml-cpath/testhisto/testhisto/utils/test_plots_utils.py

```diff
@@ -22,16 +22,17 @@ def test_plots_handler_wrong_class_names() -> None:
     assert "No class_names were provided while activating confusion matrix plotting." in str(ex)
 
 
-def test_plots_handler_slide_thumbnails_without_slide_dataset() -> None:
+@pytest.mark.parametrize("slide_plot_option", [PlotOption.SLIDE_THUMBNAIL, PlotOption.ATTENTION_HEATMAP])
+def test_plots_handler_slide_plot_options_without_slide_dataset(slide_plot_option: PlotOption) -> None:
```

Comment:
How about parametrising the `plot_options` set directly, then you can test `{THUMBNAIL}`, `{HEATMAP}`, and `{THUMBNAIL, HEATMAP}`? Currently the last case is missing.